Cython/Compiler/Nodes.py

```diff
@@ -39,7 +39,8 @@
     _py_int_types = (int, long)
 
 
-IMPLICIT_CLASSMETHODS = {"__init_subclass__", "__class_getitem__"}
+INHERITANCE_HOOK = '__init_subclass__'
```

Comment:
Please inline this constant. It's not clear from the name what it does. Seeing `"__init_subclass__"` allows me to search for it, seeing `INHERITANCE_HOOK` requires me to look up its value before I can search for it.